README.md

```diff
@@ -25,23 +25,16 @@ The project setup is documented in [project_setup.md](project_setup.md). Feel fr
 
 ## Installation
 
-To install eitprocessing from GitHub repository, do:
-
-```console
-git clone git@github.com:EIT-ALIVE/eitprocessing.git
-cd eitprocessing
-python3 -m pip install .
-```
-## Building environments
-
-You can build an environment by hand or with conda. To build with conda
-you can install all Python packages required, using conda and the
-    `environment.yml` file.
-
-  * The command for Windows/Anaconda users will be:
-     `conda env create -f environment.yml`
-
-
+Install eitprocessing as follows:
+- Create fresh environment
```

Comment:
You need to mention a minimally updated version of conda as a the older the version is, the worse it will deal with pip, and old versions will be almost incompatible with pip

Reply:
OK.